circle.yml

```diff
@@ -0,0 +1,26 @@
+dependencies:
+  pre:
+    # Replace default bundler version, which is a bit older
+    - yes y | gem uninstall bundler
+    - gem install bundler --pre
+  post:
+    - npm run build
+
+deployment:
+  staging:
+    branch: master
+    heroku:
+      appname: cs1332qna
+
+machine:
+  # Ruby version specified in .ruby-version
```

Comment:
I think that `.ruby_version` only applies to those of us using `rbenv`. We need to make sure we're standardized on what ruby version manager we're using